app2.js

```diff
@@ -51,6 +51,6 @@ function game(question, answer, element){
             }
 }
 
-for(var i=0, l = questions.length; i < l; i++){
```

Comment:
Your indentation gets bad at line 31. Check your IDE settings to make sure your tabs are set correctly